server/ip-tools.ts

```diff
@@ -44,6 +44,11 @@ export const IPTools = new class {
 
 	readonly connectionTestCache = new Map<string, boolean>();
 
+	// eslint-disable-next-line max-len
```

Comment:
tbh we should just add this to the max-len regex

Reply:
(does not have to be this PR)

Reply:
yeah i was already planning on delegating it to another pr